builder/layouts/screenshot.hbs

```diff
@@ -1,21 +1,28 @@
-{{> head }}
-
-<style>
+<!doctype html>
```

Comment:
Can we move all the doctype stuff to a partial?

Reply:
what's the benefit? we never gonna change the doctype